src/components/DesmosBlock.tsx

```diff
@@ -36,6 +39,16 @@ export function DesmosBlock({ width, height, waitForEvent, equations, disableExp
     equationsArray.forEach((str: string) => {
       calculator.setExpression({ latex: `${str}` })
     })
+
+    tablesArray.forEach((table: Array<{ variable: string, values: string[] }>) => {
+      calculator.setExpression({
+        type: 'table',
+        columns: table.map((col: { variable: string, values: string[] }) => {
+          return { latex: col.variable, values: col.values }
+        })
+      })
+    })
```

Comment:
I don't think you need to redefine the types here since TS will correctly infer them based on L19. It's minor, but this can be simplified /DRY'd up a bit to:

```javascript
    tablesArray.forEach((table) => {
      calculator.setExpression({
        type: 'table',
        columns: table.map((col) => {
          return { latex: col.variable, values: col.values }
        })
      })
    })

```